### ISSUE TITLE
[Dropdown] defects

### ISSUE THREAD

**Issue by francescogior** [buildo/react-components#108]
## Requirements
- When text input is dirty and gets in `blur` state, the `Dropdown` keeps displaying the value in the input. It should instead display the current selected option (or the placeholder 'Select...')
- When in focus, selected option or placeholder should have reduced opacity (placeholderish) to induce the user to understand he can type text to search options

**Comment by nerder** [comment]
@francescogior I'm going to fix this issue but i have some question about the second bullet point. If i understand well, you want to reduce opacity on the label of selected option. I don't very agree with this, because it can be sound confusing for the users. No real evidence of why, it just seems weird to me.

CC @FrancescoCioria

**Comment by francescogior** [comment]
look here!

[![https://gyazo.com/6ce83519af6fd2d47957a59d9c2dda7d](https://i.gyazo.com/6ce83519af6fd2d47957a59d9c2dda7d.gif)](https://gyazo.com/6ce83519af6fd2d47957a59d9c2dda7d)

**Comment by nerder** [comment]
Ok i got it. I can split this in 2 different issues closing this one?

**Comment by francescogior** [comment]
I think it's ok to work on this one for both the two tasks!

**Comment by FrancescoCioria** [comment]
About the second point I remind you that this is the default style (we're on `react-components`) --> It's likely to be overwritten in any project.

@francescogior why do you think it would useful a different default? wouldn't we change it anyway on our projects?

**Comment by francescogior** [comment]
I think that when (and only when) the `Dropdown` is `searchable` the default style should be that one:
- if an options is selected, the `:focus` style of `Dropdown` should inform the users that they can type something. This is not clear with the current component in LOL
[![https://gyazo.com/345579344f15ec56fdefd016ac7577aa](https://i.gyazo.com/345579344f15ec56fdefd016ac7577aa.gif)](https://gyazo.com/345579344f15ec56fdefd016ac7577aa)
- It can be OK for this behaviour to be overridable but I don't see why one should do it!

**Comment by nerder** [comment]
## Specs Proposal
Could i remove `valueLink` props from `DropDown`? It will not add anything and it will just add possible complications. (i.e: i have to repeat some checks for handle the same behaviour in both cases)

**Comment by nerder** [comment]
With this pr i actually solve only the first bullet point (the `onBlur` reset value issue), in order to fix even the second bug i need this to be merged first #133 , i'll provide here the code that we have to use for fix the second point:

```css
.Select-control > .Select-placeholder {
  color: #333333;
  opacity: .5;
}

.has-value > .Select-control > .Select-placeholder {
  opacity: 1;
}

.Select.dropdown.is-searchable.is-open.is-focused .Select-placeholder {
  opacity: .75;
}
```
